resources/stylesheet.css

```diff
@@ -428,5 +432,6 @@ hr {
 #changeTheme:hover {
 	cursor: pointer;
 	color: #A1C6FF;
```

Comment:
The goal is to have no hardcoded colours, so that they can be easily changed between themes. Other than that, this looks good!

Reply:
That color is for the text of the button. It should ideally always be close to white unless the button's background color is very light. 

The color of the text should depend **solely on the background color** of the button - which is decided through CSS variables when creating a new theme. So, the color of the text in the button should also be specified in the same place. 

This is an easy thing to implement: we just have to create another variable. **But then, having too many variables isn't good either.** Anyone creating a theme would be very confused. One thing we can do to help this is to provide detailed docs/guidelines for making themes for the website.

If we add more variables, it will help with the customizability of the website. So, should I go ahead and create another variable called ```--buttonText```?

Reply:
You could probably use --textSecondary, or play around with the colours that already exist. If no pre-existing colours will work, then you can create --textTertiary.

Reply:
I tried a lot of predefined colors, none of those worked. I guess I'll be going with ```--textTertiary```